ros2bag/ros2bag/verb/play.py

```diff
@@ -32,6 +32,9 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             help='size of message queue rosbag tries to hold in memory to help deterministic '
                  'playback. Larger size will result in larger memory needs but might prevent '
                  'delay of message playback.')
+        parser.add_argument(
+            '-x', '--rate', type=float, default=1.0,
```

Comment:
not a big fan of `x` here, maybe we just want to only provide the long `--rate` option?